client/webserver/site/src/html/forms.tmpl

```diff
@@ -51,12 +61,13 @@
     <hr class="dashed my-4">
   {{end}}
   <div>
-    <label for="walletPass" class="pl-1 mb-1">Password</label>
-    <input type="password" autocomplete="off" class="form-control select" id="walletPass">
+    <label for="uwClientPass" class="pl-1 mb-0">Client Password</label>
+    <div class="fs14 px-1 mb-1">Your client password is always required when performing sensitive wallet operations.</div>
```

Comment:
@buck54321 don't know if you noticed this. I added this here because it gets confusing sometimes when asked to provide a password to unlock wallet, it feels like the wallet password is what's required. 
Given your other comment about the UX disadvantage of showing tips like this everytime the form is used, I'm rethinking this. Should the change from `Password` to `Client Password` suffice?

Reply:
Definitely better, and it needs to be standardized. We call it the "App password" in **rpcserver**.

Reply:
`Client Password` or `App Password`. Difficult to prefer one over the other.
Looking at it from the user's PoV, `App Password` maybe... because `Client Password` suggests the existence of a `Server Password`, which a typical user may not know about even though it does exist. Users are familiar with Apps and App Passwords, I think.

Endorse change `Client Password` to `App Password`?

Reply:
Sounds good to me.